tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/js/operator-finder/operator-finder.js

```diff
@@ -126,7 +127,80 @@ define(['jquery', 'lodash', 'log', 'handlebar', 'designViewUtils', 'app/source-e
         };
 
         /**
-         * Flattens the metadata structure into an array ti reduce the search complexity.
+         * replacing the pipeline and newline character for md conversation.
+         * @param data string variable
+         * @returns {""} retrun string data
+         */
+        var pipelineNewlineEscape = function (data) {
```

Comment:
```suggestion
        var normaliseString = function (data) {
```

Reply:
960249df47be3bbdc76d0016d7e1dd9ac324ac05 this commit fix above request change